doc/getting_started/installation_linux.rst

```diff
@@ -66,6 +66,11 @@ Install the required packages in a Fedora host system with:
 	 libstdc++-static python3-ply ncurses-devel \
 	 python-yaml python2 dfu-util
 
+.. note::
```

Comment:
This really has nothing to do with Zephyr, we can't be adding notes for every compiler out there. icx has some requirements, so it is an icx issue. Adding the note here might automatically guarantee that icx would work on Fedora 25, which we as the zephyr project can't confirm or support.